database/things/postgresql.pldb

```diff
@@ -24,6 +24,7 @@ features
   /* A comment
   */
  hasSemanticIndentation false
+ hasCaseInsensitiveIdentifiers false
```

Comment:
Again, https://en.wikipedia.org/wiki/Case_sensitivity#In_programming_languages says:
> Others are case-insensitive (i.e., not case-sensitive), such as... [SQL](https://en.wikipedia.org/wiki/SQL) (for the syntax)

According to https://en.wikipedia.org/wiki/PostgreSQL#Standards_compliance, it seems to be case-insensitive.

Reply:
@celtic-coder , same scenario as php but most of the websites suggest its case sensitive. What you suggest? I will change accordingly.

Reply:
I've written lots of SQL code in both the Microsoft and Oracle varieties and on these platforms it is case-insensitive. I would imagine that PostgreSQL would be the same. The "Standards Compliance" section of the Wikipedia article would seem to indicate this also.